content/tutorials/encryption-realtime.textile

```diff
@@ -0,0 +1,618 @@
+
+  Include the code below just before your closing your HTML @</html>@ tag.
+
+  ```[html]
+    <!-- Include the latest Ably Library  -->
+    <script src="https://cdn.ably.io/lib/ably.min-1.js"></script>
+
+    <!-- Instantiate the Ably library  -->
+    <script type="text/javascript">
+      var realtime = new Ably.Realtime('INSERT-YOUR-API-KEY-HERE');
+    </script>
+  ```
+
+  **What have we setup in @publisher.html@ so far?**
+
+  # We have added a text box to accept message input from user.
+  # We have provided a button to encrypt and publish the input message. We'll see how in the following step.
+  # We added Ably library and configured it with our API key. Remember to replace @'INSERT-YOUR-API-KEY-HERE'@ with your Ably API key from Step #1
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/encryption-js-step2
+
+blang[nodejs].
+  The Ably Realtime and REST library is available as an "NPM module":https://www.npmjs.com/package/ably.
+  Simply add it to the @package.json@ file of your NodeJs project.
+
+  ```[json]
+  {
+      ...
+      "dependencies": {
+          "ably": ">=1.1"
+          ...
+      }
+  }
+  ```
+
+  and run:
+
+  ```[sh]
+  npm install
+  ```
+
+  or install it using @npm install@:
+
+  ```[sh]
+  npm install ably
+  ```
+
+  The client library must be instanced with the API key you copied in Step 1. API keys used with "basic authentication":/core-features/authentication#basic-authentication for your own servers is generally preferred, however clients running on insecure devices should always use the "token authentication scheme":/core-features/authentication#token-authentication instead. In this example, we use an API key for simplicity.
+
+  Add the following to @constants.js@ to define constants we'll use throughout our project:
+
+  ```[nodejs]
+    const API_KEY = 'INSERT-YOUR-API-KEY-HERE'; /* Add your API key here */
+    if (API_KEY.indexOf('INSERT') === 0) {
+        throw('Cannot run without an API key. Add your key to this file.');
+    }
+
+    module.exports = {
+        API_KEY: API_KEY
+    };
+  ```
+
+  Here we defined just one constant @API_KEY@. Remember to replace @'INSERT-YOUR-API-KEY-HERE'@ with your Ably API key from Step #1
+
+  Now, let's create a @publisher.js@ with following content:
+
+  ```[nodejs]
+    const Ably = require('ably');
+    const constants = require('./constants');
+
+    /* Instance the Ably library */
+    const realtime = new Ably.Realtime({key: constants.API_KEY});
+  ```
+
+  Here, we're simply importing @Ably@ and @constants.js@.
+  Then we define a @Realtime@ instance, which we'll use to obtain a channel in the next steps.
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/encryption-nodejs-step2
+
+h2(#publish-messages). Step 3 - Publish encrypted messages to a channel
+
+Let's see how to leverage the encryption API to publish encrypted messages to a channel.
+
+blang[java].
+
```

Comment:
Remove new line after `blang` throughout the document. The compilation fails otherwise. As mentioned, please check if it renders locally properly before sending it out for review.